R/rsynccli.R

```diff
@@ -36,5 +36,23 @@ rsynccli <- function(file, to, includes = NULL, excludes = NULL, args = "-rltvx"
     to
   )
 
-  system(command, intern = intern, wait = TRUE, ignore.stdout = FALSE, ignore.stderr = FALSE)
+  status <- system(command, intern = intern, wait = TRUE, ignore.stdout = FALSE, ignore.stderr = FALSE)
+  checkForStatus(status)
+}
+
+checkForStatus <- function(result) {
+  if (is.character(result) && is.null(attr(result, "status"))) {
+    sys_output <- result
+    return(sys_output)
+  }
```

Comment:
Bekommen wir den sys_output nicht ohnehin, wenn wir mit verbose = TRUE arbeiten?

Reply:
Vielleicht verstehe ich dich falsch. Das Problem ist das die `system` Funktion manchmal den Status Code zurück gibt und manchmal den Output des bash Kommandos. Dies ist ja auch gleichzeitig unser Rückgabewert der rsync funktion. Falls wir da aneinander vorbei reden, sag mir gerne nochmal Bescheid.

Reply:
Wenn wir die Funktion mit verbose = TRUE aufrufen, wird der Inhalt von stdout in die Konsole geschrieben.